api/functions/requirements.txt

```diff
@@ -2,4 +2,5 @@ google-cloud-storage
 google-cloud-pubsub
 google-auth
 firebase-admin
-functions-framework==3.*
\ No newline at end of file
+functions-framework==3.*
```

Comment:
Any reason not to pin these at a specifc version? Upstream changes can be a nightmare (and hard to notice, especially in a lambda-like environment)

Reply:
I actually added this back in for no reason- we're not using this as of now so I've taken it out but have seen that the rest of our requirements.txt file is full of unpinned dependencies, so I'm making an issue to remedy this but not addressing it in this PR.